Formula/notmuch.rb

```diff
@@ -11,12 +12,13 @@ class Notmuch < Formula
     sha256 "6e5950e4159217dc865277576e3c75875269a286fb26a8e3a3e52987e651717f" => :mavericks
   end
 
+  option "without-python", "Build without python support"
+
   depends_on "pkg-config" => :build
   depends_on "gmime"
   depends_on "talloc"
   depends_on "xapian"
   depends_on :emacs => ["21.1", :optional]
-  depends_on :python => :optional
```

Comment:
Doesn't that pull in `python` as a dependency? The bindings work with the system python.